internal/controllers/machineset/machineset_controller.go

```diff
@@ -561,19 +568,17 @@ func (r *Reconciler) syncMachines(ctx context.Context, s *scope) (ctrl.Result, e
 			return ctrl.Result{}, errors.Wrapf(err, "failed to get InfrastructureMachine %s %s",
 				updatedMachine.Spec.InfrastructureRef.Kind, klog.KRef(updatedMachine.Namespace, updatedMachine.Spec.InfrastructureRef.Name))
 		}
-		// Cleanup managed fields of all InfrastructureMachines to drop ownership of labels and annotations
-		// from "manager". We do this so that InfrastructureMachines that are created using the Create method
-		// can also work with SSA. Otherwise, labels and annotations would be co-owned by our "old" "manager"
-		// and "capi-machineset" and then we would not be able to e.g. drop labels and annotations.
-		labelsAndAnnotationsManagedFieldPaths := []contract.Path{
-			{"f:metadata", "f:annotations"},
-			{"f:metadata", "f:labels"},
-		}
-		if err := ssa.DropManagedFields(ctx, r.Client, infraMachine, machineSetManagerName, labelsAndAnnotationsManagedFieldPaths); err != nil {
-			return ctrl.Result{}, errors.Wrapf(err, "failed to update machine: failed to adjust the managedFields of the InfrastructureMachine %s", klog.KObj(infraMachine))
+		// Drop managedFields for manager:Update and capi-machineset:Apply for all objects created with CAPI <= v1.11.
```

Comment:
nit
```suggestion
		// Drop managedFields for manager: Update and capi-machineset: Apply for all objects created with CAPI <= v1.11.
```

Reply:
I intentionally didn't add a space there. I think it's easier to see that this is basically a "full" fieldManager name (manager:operation) without a space in the middle